cdap-explore-jdbc/src/main/java/co/cask/cdap/explore/jdbc/ExploreDriver.java

```diff
@@ -65,17 +65,26 @@ public Connection connect(String url, Properties info) throws SQLException {
     ConnectionParams params = parseConnectionUrl(url);
 
     String authToken = null;
+    String namespace;
 
     List<String> tokenParams = Lists.newArrayList(params.getExtraInfos().get(ConnectionParams.Info.EXPLORE_AUTH_TOKEN));
     if (tokenParams != null && !tokenParams.isEmpty() && !tokenParams.get(0).isEmpty()) {
       authToken = tokenParams.get(0);
     }
 
+    List<String> namespaceParams =
+      Lists.newArrayList(params.getExtraInfos().get(ConnectionParams.Info.NAMESPACE));
+    if (namespaceParams != null && !namespaceParams.isEmpty() && !namespaceParams.get(0).isEmpty()) {
```

Comment:
namespaceParams cannot be null, since we are just creating it above. Can we remove the check for null?